v3/tasks/20250530-v3-cow-swap-fee-burner/input.ts

```diff
@@ -0,0 +1,37 @@
+import { Task, TaskMode } from '@src';
+
+export type CowSwapFeeBurnerDeployment = {
+  ProtocolFeeSweeper: string;
+  AppDataHash: string;
+  ComposableCow: string;
+  CowVaultRelayer: string;
+  InitialOwner: string;
+  Version: string;
+};
+
+// NOTE: AppDataHash was encoded here: https://explorer.cow.fi/appdata?tab=encode
+// - App code: `CowSwapFeeBurner-V1`
```

Comment:
Would it be worth changing this app data hash?

Reply:
I think it is not a big deal, but I will ask @gosuto-inzasheru